go.mod

```diff
@@ -6,11 +6,12 @@ require (
 	github.com/alexflint/go-arg v1.3.0
 	github.com/cloudboss/ofcourse v0.2.2
 	github.com/gertd/wild v0.0.2-0.20200810044159-389fd0bed676
-	github.com/google/go-cmp v0.5.4
+	github.com/google/go-cmp v0.5.8
 	github.com/sasbury/mini v0.0.0-20181226232755-dc74af49394b
+	gotest.tools/v3 v3.3.0
 )
 
 require (
 	github.com/alexflint/go-scalar v1.0.0 // indirect
-	github.com/stretchr/testify v1.7.0 // indirect
+	github.com/stretchr/testify v1.7.2 // indirect
```

Comment:
"indirect" means that it is required by a dependency, in this case ofcourse. So it will go away at the end of this series of PRs.